src/SimpleThings/EntityAudit/EventListener/LogRevisionsListener.php

```diff
@@ -469,7 +469,7 @@ private function getHash($entity)
         return implode(
             ' ',
             array_merge(
-                array(get_class($entity)),
+                array(\Doctrine\Common\Util\ClassUtils::getRealClass(get_class($entity))),
```

Comment:
This can be simplified by using `ClassUtils::getClass($entity)` instead.

Ref: https://github.com/doctrine/common/blob/v2.7.1/lib/Doctrine/Common/Util/ClassUtils.php#L56-L59

Also, you may import the namespace to get a cleaner code.